packages/relay-runtime/store/OperationExecutor.js

```diff
@@ -1558,7 +1558,7 @@ function validateOptimisticResponsePayload(
 }
 
 const isPerformanceNowAvailable =
-  global.performance != null && typeof global.performance.now === 'function';
+  global?.performance != null && typeof global.performance.now === 'function';
```

Comment:
Thanks @mellson! This got moved to https://github.com/facebook/relay/blob/master/packages/relay-runtime/util/withDuration.js, could you rebase and update your PR?

Reply:
@poteto sure thing!

Reply:
Hi @poteto I made a mess of the rebasing 🥴 So I recreated this much cleaner here https://github.com/facebook/relay/pull/3525